### DISCUSSION TITLE
Ubuntu PPA?

### DISCUSSION THREAD

**Discussion by AbnormalPoof** [FunkinCrew/Funkin#1699] [Ideas]
just so people dont go through the trouble of compiling for linux

**Comment by htv04** [comment]
The Linux build here works fine for me on Ubuntu: https://ninja-muffin24.itch.io/funkin

Although it is one version outdated since it runs on v0.2.7 instead of v0.2.7.1

**Reply by AbnormalPoof** [reply]
i know, but maybe have a ppa so people can do shit like `sudo apt install Funkin` and run the game instead of cding into the folder and executing `./Funkin`

**Comment by ninjamuffin99** [comment]
I need to dig into linux stuff to really figure out it all. Are stuff like PPA's required? I compiled the game using Manjaro Linux, so I'm not sure if different distros use different formats for binary files or whatev. I'll dig into it, figure out a nice little one click method so that people don't need to fuk around with terminal at the very least.

**Reply by htv04** [reply]
You can try creating a Flatpak, which works on any distro.

**Reply by AbnormalPoof** [reply]
flatpak works too, didnt think of that
